apps/vx-mark/frontend/src/config/types.ts

```diff
@@ -6,78 +6,14 @@ import {
   Contests,
   Election,
   ElectionDefinition,
-  MachineId,
   MsEitherNeitherContest,
   OptionalVote,
   OptionalYesNoVote,
   PrecinctId,
   VotesDict,
   YesNoContest,
 } from '@votingworks/types';
-import { z } from 'zod';
-
-// App
-export const PrintOnly = {
-  key: 'PrintOnly',
-  productName: 'VxPrint',
-  isMark: false,
-  isPrint: true,
-} as const;
-export const MarkOnly = {
-  key: 'MarkOnly',
-  productName: 'VxMark',
-  isMark: true,
-  isPrint: false,
-} as const;
-export const MarkAndPrint = {
-  key: 'MarkAndPrint',
-  productName: 'VxMark',
-  isPrint: true,
-  isMark: true,
-} as const;
-export type AppMode = typeof MarkOnly | typeof PrintOnly | typeof MarkAndPrint;
-export type AppModeKeys = AppMode['key'];
-export const AppModeKeysSchema: z.ZodSchema<AppModeKeys> = z.union([
-  z.literal(PrintOnly.key),
-  z.literal(MarkOnly.key),
-  z.literal(MarkAndPrint.key),
-]);
-
-export type ScreenOrientation = 'portrait' | 'landscape';
-
-export interface MachineConfig {
-  machineId: string;
-  appMode: AppMode;
-  codeVersion: string;
-  screenOrientation: ScreenOrientation;
-}
-
-export interface MachineConfigResponse {
-  machineId: string;
-  appModeKey: AppModeKeys;
-  codeVersion: string;
-  screenOrientation: ScreenOrientation;
-}
-export const MachineConfigResponseSchema: z.ZodSchema<MachineConfigResponse> =
-  z.object({
-    machineId: MachineId,
-    appModeKey: AppModeKeysSchema,
-    codeVersion: z.string().nonempty(),
-    screenOrientation: z.union([z.literal('portrait'), z.literal('landscape')]),
-  });
-
-export function getAppMode(key: AppModeKeys): AppMode {
-  switch (key) {
-    case PrintOnly.key:
-      return PrintOnly;
-    case MarkOnly.key:
-      return MarkOnly;
-    case MarkAndPrint.key:
-      return MarkAndPrint;
-    default:
-      throw new Error(`unknown app mode: ${key}`);
-  }
-}
+import { MachineConfig } from '@votingworks/vx-mark-backend';
```

Comment:
`import type` here?

Reply:
I don't think I can do import type because MachineConfig is an interface

Reply:
`interface` and `type` are nearly equivalent, and you can `import type` with an `interface`.

Reply:
oh.. when I try to add type eslint just autoremoves it so I assumed it didn't work with interfaces but I'll play around more

Reply:
I think it'll work if you disable the relevant eslint rule!

https://github.com/votingworks/vxsuite/blob/f0147e2eee46d5fe54be6617ecc62499b1b7bf42/apps/vx-scan/frontend/src/app.tsx#L9-L10

I also recently got tripped up by the auto-removal

Reply:
oh duh thanks